ui/app/components/ui/searchable-item-list/item-list/item-list.component.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import classnames from 'classnames'
+
+export default function SearchableItemList ({
+  results = [],
+  onClickItem = null,
+  Placeholder = null,
+  listTitle = '',
+  maxListItems = 6,
+  searchQuery = '',
+}) {
+  return results.length === 0
+    ? <Placeholder searchQuery={searchQuery} />
+    : (
+      <div className="searchable-item-list">
+        {listTitle && (
+          <div className="searchable-item-list__title">
+            { listTitle }
+          </div>
+        )}
+        <div className="searchable-item-list__list-container">
+          {
+            results.slice(0, maxListItems)
+              .map((result, i) => {
+                const {
+                  backgroundImageUrl,
+                  selected,
+                  disabled,
+                  primaryLabel,
+                  secondaryLabel,
+                  rightPrimaryLabel,
+                  rightSecondaryLabel,
```

Comment:
Are these used anywhere currently (aside from in Storybook)? Or are they solely for future use?

Reply:
For future use